composer.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "woocommerce/wc-api-dev",
```

Comment:
Yeah, there are dashes in the others here: https://github.com/Automattic/wpcomsh/blob/master/composer.json

I'm not worried about it, but thanks for being thorough!